src/test/compile-fail/private-variant-reexport.rs

```diff
@@ -9,19 +9,19 @@
 // except according to those terms.
 
 mod m1 {
-    pub use ::E::V; //~ ERROR variant `V` is private, and cannot be reexported
+    pub use ::E::V; //~ ERROR variant is private and cannot be reexported
 }
 
 mod m2 {
-    pub use ::E::{V}; //~ ERROR variant `V` is private, and cannot be reexported
+    pub use ::E::{V}; //~ ERROR variant is private and cannot be reexported
```

Comment:
What is the stderr output for this file? I want to make sure that it is pointing at the `V` path segment exclusively (here and elsewhere in this file), otherwise change the `msg` above to include the identifier as it was before to avoid confusion, as if the output points at the entire `use` statement (which is my understanding from reading the code) it could be referring to `E` or `V` (and it gets worse the more path segments there are).

Reply:
It was

```
zmd@MaximumLikelihood:~/Code/rust$ ./build/x86_64-unknown-linux-gnu/stage1/bin/rustc src/test/compile-fail/private-variant-reexport.rs
error: variant is private and cannot be reexported
  --> src/test/compile-fail/private-variant-reexport.rs:12:13
   |
12 |     pub use ::E::V; //~ ERROR variant is private and cannot be reexported
   |             ^^^^^^
...
27 | enum E { V }
   | ------ help: consider making the enum public: `pub enum E`

error: variant is private and cannot be reexported
  --> src/test/compile-fail/private-variant-reexport.rs:16:19
   |
16 |     pub use ::E::{V}; //~ ERROR variant is private and cannot be reexported
   |                   ^

error: variant is private and cannot be reexported
  --> src/test/compile-fail/private-variant-reexport.rs:20:22
   |
20 |     pub use ::E::V::{self}; //~ ERROR variant is private and cannot be reexported
   |                      ^^^^

error: enum is private and its variants cannot be reexported
  --> src/test/compile-fail/private-variant-reexport.rs:24:13
   |
24 |     pub use ::E::*; //~ ERROR enum is private and its variants cannot be reexported
   |             ^^^^^^

error: aborting due to 4 previous errors
```
(Span is including the `E::` for the non-brace case; can add variant name back into the message.)